bin/connect-distributed.sh

```diff
@@ -26,6 +26,10 @@ if [ "x$KAFKA_LOG4J_OPTS" = "x" ]; then
     export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$base_dir/../config/connect-log4j.properties"
 fi
 
+if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
+  export KAFKA_HEAP_OPTS="-Xms256M -Xmx2G"
+fi
+
```

Comment:
Should we also do this in `bin/connect-standalone.sh`?

Reply:
But, yes, IMO these are the right settings to make and the correct way to do it.

Reply:
Yeah, it should be better to keep both these settings consistent. Let me update this PR. Thanks, @rhauch.

Reply:
Should this be accompanied with a log message on output. 

i.e. 

"No JVM memory settings set: Sticking to the Default of Min : 256 mb Max: 2048mb."

The JVM memory limitations/settings weren't super clear to me when I started provisioning clusters.

Reply:
@bencoughlan: we don't want to output anything in the shell scripts (as free text in shell scripts might corrupt the application log stream). An alternative would be to add some documentation on what the default heap settings are, and how we can change them while running the connect scripts. Thoughts?

Reply:
A quick addition to my comment: All the jvm args are already being printed when the connect workers start. It should be the first or second log message in the worker. Hope this helps.

Reply:
I mean, yeah, that sounds like a good idea. 

even if it was in the properties file be default over each value 

#Default = X if not set. 

etc. 

Does that make sense? 

This JVM setting, in particular, wasn't overly obvious.